src/data/docs/documentation.md

```diff
@@ -5,257 +5,232 @@ description: How to write Unison documentation and associate it with your code
+    nonsense     : Text
+    nonsense.doc : Doc2
 ```
 
-The documentation value `myFunction.doc` is now linked to the `myFunction` definition in the codebase and we can check this using the `links` command:
+## Viewing Docs 
+To read the documentation associated with any term or type declaration, use the `docs` or `display` command in the UCM. A command taking the form `docs myTermName` will look for a term called `myTermName.doc` in the file or codebase, so we do not need to explicitly link our docs to our code if we respect that naming convention. 
 
 ```ucm
-.> links myFunction
+.> docs nonsense
 
-  1. myFunction.doc : Doc
+  `"Colorless green ideas sleep furiously."` - Noam Chomsky, is a grammatically well formed, but non-sensical
+  sentence
 ```
 
-This demonstrates the Unison codebase's system of *links*, which allows you to link documentation to code. This system also lets you link all kinds of metadata other than documentation, but for now let's concentrate on docs.
-
-## Viewing the docs for a definition
-
-We can ask `ucm` to display the documentation for a given definition using the `docs` command:
+To view the implementation of our documentation, we can use the `view` command. 
 
 ```ucm
-.> docs myFunction
+.> view nonsense.doc
 
-  This function multiplies its argument by two.
+  nonsense.doc : Doc2
+  nonsense.doc =
+    {{
+    @eval{ nonsense } - {{ author }}, is a grammatically well formed, but non-sensical sentence
+    }}
 ```
 
-## The documentation syntax in detail
-
-The syntax `[:` opens a documentation block, and `:]` closes it. Everything in between is documentation, and the whole block is a Unison value of type `builtin.Doc`. Inside a documentation block, you can use the following special syntax elements.
+So - we know how to define a simple doc, link it to a definition, and render it in the console. Let's explore some of the richer formatting cabilities of Unison docs. 
 
-### Hyperlinks to other definitions
-
-You can create a hyperlink to a Unison definition named `myDef` in your codebase with `@myDef`. For example we could have the documentation for `myFunction` reference that function itself:
+# Text Formatting and Linking
 
+Let's look at the following docs example to see more of the documentation type's powers: 
 ```unison
-myFunction.doc =
-  [: @myFunction multiplies its argument by two. :]
-```
+{{
+  PartOfSpeech describes the nine ways a word can function in a phrase. Defined as: 
 
-The hyperlink is based on a hash of `myFunction` rather than its name, so if it's ever renamed or moved, the link will still work.
+  @source{type PartOfSpeech}
 
-### Insert the source code of another definition
+  {{furtherExposition}}
 
-You can include the full source code of any definition in your codebase using `@[source]` followed by the name of the definition. For example, to include the source code of `myFunction` in its own documentation:
+  {{examples}}
+}}
+unique type PartOfSpeech Text = 
```

Comment:
I feel that this example can trip up the reader. Are parts of speech important for documentation? Do I, as a doc author, need to know about this stuff? On reflection it's clear that I don't, but it would be nice to obviate that reflection. I think the domain of these examples (linguistics) is too close to the topic under discussion (documentation).